packages/core/src/command/consolidated/search/fhir-resource/ingest-lexical.ts

```diff
@@ -89,7 +90,17 @@ async function getConsolidatedBundle({
 }): Promise<Bundle<Resource>> {
   const { log } = out(`getConsolidatedBundle - cx ${cxId}, pt ${patientId}`);
 
-  const consolidated = await getConsolidatedFile({ cxId, patientId });
+  const consolidated = await executeWithRetries(() => getConsolidatedFile({ cxId, patientId }), {
+    shouldRetry: (res, error: unknown) => {
+      if (!res) return true;
+      if (!error) return false;
+      if (!isRetriableError(error)) return false;
+      return true;
+    },
+    initialDelay: 1_000,
+    maxAttempts: 3,
+    log,
+  });
```

Comment:
`getConsolidatedFile` already has retries through S3Utils, and the implementations look pretty similar - so no need for this here?

Reply:
The idea is to retry in 404, since we get missing bundle sometimes